testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js

```diff
@@ -0,0 +1,119 @@
+    });
+
+    QUnit.test("element should have buttonGroup inside", assert => {
+        const buttonGroup = this.instance._buttonGroup;
+        assert.strictEqual(buttonGroup.NAME, "dxButtonGroup", "buttonGroup rendered");
+        assert.strictEqual(buttonGroup.option("selectionMode"), "none", "selection should be disabled");
+        assert.strictEqual(buttonGroup.option("stylingMode"), "outlined", "styling mode should be outlined");
+
+        const buttonGroupItems = buttonGroup.option("items");
+        assert.strictEqual(buttonGroupItems.length, 2, "2 buttons are rendered");
+        assert.strictEqual(buttonGroupItems[0].icon, "default", "empty icon is correct");
+        assert.strictEqual(buttonGroupItems[1].icon, "chevrondown", "dropdown icon is correct");
+    });
+
+    QUnit.test("popup should not be rendered if deferRendering is false", assert => {
+        assert.strictEqual(getPopup(this.instance), undefined, "popup should be lazy rendered");
+        assert.strictEqual(getList(this.instance), undefined, "list should be lazy rendered");
+    });
+
+    QUnit.test("popup and list should be rendered on init when deferRendering is false", assert => {
+        const dropDownButton = new DropDownButton("#dropDownButton", { deferRendering: false });
+        const popup = getPopup(dropDownButton);
+
+        assert.strictEqual(popup.NAME, "dxPopover", "popup has been rendered");
+        assert.strictEqual(getList(dropDownButton).NAME, "dxList", "list has been rendered");
+        assert.strictEqual(popup.option("target"), getButtonGroup(dropDownButton).element(), "popup has correct target");
+        assert.ok(popup.option("closeOnOutsideClick"), "popup should be closed on outside click");
+    });
+});
+
+QUnit.module("list integration", {}, () => {
+    QUnit.test("list should be displayed correctly without data expressions", assert => {
+        const dropDownButton = new DropDownButton("#dropDownButton", {
+            items: ["Item 1"],
+            deferRendering: false
+        });
+        const list = getList(dropDownButton);
+        const $listItem = list.itemElements();
+
+        assert.strictEqual($listItem.text(), "Item 1", "displayExpr works");
+        assert.strictEqual(list.option("keyExpr"), null, "valueExpr is 'this'");
+    });
+
+    QUnit.test("data expressions should work with dropDownButton", assert => {
+        const dropDownButton = new DropDownButton("#dropDownButton", {
+            items: [{ key: 1, name: "Item 1", icon: "box" }],
+            valueExpr: "key",
+            displayExpr: "name",
+            deferRendering: false
+        });
+
+        const list = getList(dropDownButton);
+        const $listItem = list.itemElements();
+
+        assert.strictEqual($listItem.text(), "Item 1", "displayExpr works");
+        assert.strictEqual(list.option("keyExpr"), "key", "valueExpr works");
+        // assert.strictEqual($listItem.find(".dx-icon-box").length, 1, "item icon works");
```

Comment:
Why is this line commented out?

Reply:
This line was commented before the PR about displayExpr option in the list is merged